yarn.lock

```diff
@@ -0,0 +1,6179 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Yes this basically ensure that `yarn` resolves package versions the same for everyone installing osrm. One advantage of the `yarn` approach with the `lock` file is to enable caching, e.g. on Travis.